src/tutorial/modal.component.tsx

```diff
@@ -39,20 +40,21 @@ const TutorialModal = ({ open, onClose }) => {
 
   return (
     <React.Fragment>
-      <ModalHeader closeModal={onClose} title={t('tutorial', 'Tutorial')}>
+      <ModalHeader className={styles.modalHeader} closeModal={onClose} title={t('tutorial', 'Tutorial')}>
         <p className={styles.description}>
           {t('modalDescription', 'Find walkthroughs and video tutorials on some of the core features of OpenMRS.')}
         </p>
       </ModalHeader>
-      <ModalBody className={styles.tutorialModal}>
+      <ModalBody className={styles.tutorialModal} hasScrollingContent >
         <ul>
           {tutorials.map((tutorial, index) => (
             <li className={styles.tutorialItem} key={index}>
               <h3 className={styles.tutorialTitle}>{tutorial.title}</h3>
               <p className={styles.tutorialDescription}>{tutorial.description}</p>
-              <Button kind="ghost" onClick={() => handleWalkthroughClick(index)}>
+              <Link className={styles.tutorialLink} onClick={() => handleWalkthroughClick(index)} renderIcon={() => 
+                <ArrowRight aria-label="Arrow Right" />}>
```

Comment:
@jayasanka-sack do we need to keep an `Arrow Right` here?
It's not defined in the design though.

@IamMujuziMoses Also please make the cursor to pointer

Reply:
I think the cursor is set to pointer by default for the link component. Please confirm.